packages/react/src/date-picker/DatePicker.stories.tsx

```diff
@@ -73,6 +73,12 @@ export const Block: Story = {
 	},
 };
 
+export const AlternativeDateFormat: Story = {
```

Comment:
Yep, creates the same Alternative Date Format story in Storybook for both DatePicker and DateRangePicker

Reply:
It is better to have a separate file with common date variables that are exported and shared by both date components rather than having the variables duplicated across 2 files. It is much better for maintainability that way.

Reply:
While I agree DRY is good principle in general, in this case it would add unnecessary overhead maintaining a separate file to have 1 story that is identical between the 2 types of date pickers

Reply:
I don't actually know what a "story" is in the context of the Ag Design System

Reply:
No worries. We use [Storybook](https://storybook.js.org/) as an isolated component development environment. Each component has stories, which are basically ways to show important variations of props and state. Here's the story related to this addition https://design-system.agriculture.gov.au/pr-preview/pr-1563/storybook/index.html?path=/story/forms-daterangepicker--alternative-date-format

Reply:
Ok so it looks like Storybook is a sort of unit testing tool.
Ok, I agree. In this context the duplication of variables makes more sense than exporting them out into a separate file.